File: Python/백준/Bronze/21197. Stopwatch/Stopwatch.py
a = int(input())
b = 0
c = 0 #정지 상태
for i in range(a):
    b = int(input())-b
    if c==0:
        c=1 #실행 상태
    else:
        c=0
if c==1:
    print('still running')
else:
    print(b)

File: Java/백준/Bronze/21197. Stopwatch/Stopwatch.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        if(a%2==1){
            System.out.println("still running");
        }
        else{
            int s = 0;
            for (int i=0;i<a/2;i++){
                s+= -in.nextInt()+in.nextInt();
            }
            System.out.println(s);
        }
    }
}